README.md

```diff
@@ -178,9 +178,16 @@ and exclude any alpha or beta versions.
 
 ## Release a New Live Demo Version
 
-Increment `currentVersion` in the gradle.properties file, commit, and push to the `main` branch.
-This will start a build on Jenkins, which will automatically create a tag. Creating the tag will start another build on Travis.
-The compiled, bundled, and ready to run `acrolinx-sidebar-demo-java.zip` will be uploaded to the release assets.
+Create a task branch, commit your changes.
+Increment `currentVersion` in the gradle.properties file.
+
+Create a tag via using the following commands:
+```bash
+git tag v[major].[minor].[fix] (etc. v0.11.0)
+git push origin tag v[major].[minor].[fix]
+```
+
+Github will create the specified tag on the remote repository and it will build and make a release when merged with the main branch there. Artifacts are automatically generated.
```

Comment:
We had a similar discussion last week with Abhijeet when releasing the sidebar-sdk-java. I think there is a small flaw, since GitHub would build this twice, one when you create the new MR and one more time when you merge it to the main branch. Not sure what would happen when the same release assets are created and uploaded twice?

For the sidebar-sdk-jav we therefore did it like this:
- create an MR where we updated the currentVersion property
- GitHub builds this branch and releases the artifacts
- If all went well, close the MR and discard the branch without merging
- then create a new MR and update the currentVersion to "x.y.z-SNAPSHOT" (whatever the new snapshot version after the release should be). Get an approval and merge to main.

Maybe we should do the same here for this repo?

Reply:
Hi Thilo we can handle it as you suggest if it works. I made the proposal just to update the way we tested that it works currently as we don’t use Travis anymore.
We can just close this PR and you can update it the way you think it works best. I have no objection there.

Reply:
@Thilo-Acrolinx is right. We have been thinking of how to publish the artifacts only once. 
Also, this looks like a good feature @nixor, maybe we can re-open the MR once we figure out a practical way of publishing the artifacts only once.

To clarify, who owns the GitHub Bot (is it GitHub)? Or this is a custom GitHub bot that you guys created.

Reply:
@willynganga @Thilo-Acrolinx  Somebody created a while back.

In the case that you plan to improve on the current process, sure you can reopen another PR to fix this in the proper way you deem fit.

But as it stand I felt the need to update the read me. At least for the moment this is a way that it works. As we no longer use Travis anymore.

Therefore i think we should approve at least this readme edit changes. And when you have time to look into the future and decide how it should be properly done, then we can make another change.